src/clientStubFromSlash.ts

```diff
@@ -3,10 +3,15 @@ import * as Url from "url-parse";
 import { DgraphClientStub } from "./clientStub";
 
 const PORT = "443";
+/**
+ * @deprecated since v21.3 and will be removed in v21.07 release.
```

Comment:
We can add link to refer to https://discuss.dgraph.io/t/regarding-slash-cloud-dgraph-endpoints-in-the-clients/13492 as well